roles/katello_pulp3/tasks/main.yml

```diff
@@ -0,0 +1,130 @@
+---
+- name: create static dir
+  file:
+    path: '/var/lib/pulp/static/'
+    state: directory
+    mode: '777'
+
+- include_role:
+    name: "{{ item }}"
+  with_items:
+    - pulp-database
+    - pulp-workers
+    - pulp-resource-manager
+    - pulp-content
+  vars:
+    pulp_default_admin_password: password
+    pulp_requirements_dir: "https://raw.githubusercontent.com/pulp/pulpcore/master/"
+    pulp_plugin_source_dir: "git+https://github.com/pulp/pulpcore-plugin.git#egg=pulpcore-plugin"
+    pulp_source_dir: "git+https://github.com/pulp/pulpcore.git#egg=pulpcore"
+    pulp_install_plugins:
+      pulp-file:
+        app_label: "file"
+        source_dir: "git+https://github.com/pulp/pulp_file.git#egg=pulp-file"
+      pulp-docker:
+        app_label: "docker"
+        source_dir: "git+https://github.com/pulp/pulp_docker.git#egg=pulp-docker"
+      pulp-ansible:
+        app_label: "ansible"
+        source_dir: "git+https://github.com/pulp/pulp_ansible.git#egg=pulp-ansible"
+    pulp_settings:
+      secret_key: secret
+      content_host: "{{ ansible_fqdn }}"
+      databases:
```

Comment:
There are 2 config settings needed for ansible collections: https://github.com/pulp/pulp_ansible/blob/90f8c3fcd974682a43578ca47fee9cc8902007a1/.travis/ansible_playbook.yml

Would it make sense to add it here. The refresh_distribution task fails without these, specifically the ansible_api_hostname setting:
```
ansible_api_hostname: 'http://localhost:24817'
ansible_content_hostname: 'http://localhost:24816/pulp/content'
```